src/api/spec/factories/maintained_project.rb

```diff
@@ -6,5 +6,10 @@
   factory :maintained_project do
     project
     maintenance_project
+
+    transient do
+      project_name { 'openSUSE:Leap:15.4' }
+      package_name { 'bash' }
+    end
```

Comment:
Where are you using this factory?

Be careful, the naming is confusing. This factory represents the [MaintainedProject model](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/maintained_project.rb), which simply associates a project (usually an Update Project like `openSUSE:Leap:15.4:Update`) with the maintenance project (`openSUSE:Maintenance`).

So I'd say setting project and package names through transient makes little sense.

```suggestion
```

Reply:
Ouch, that's true. I'm not using that.